src/app_service/common/network_constants.nim

```diff
@@ -207,6 +207,12 @@ var NODE_CONFIG* = %* {
     "VerifyTransactionURL": "https://mainnet.infura.io/v3/" & INFURA_TOKEN_RESOLVED,
     "BandwidthStatsEnabled": true
   },
+  "HTTPEnabled": true,
+  "HTTPHost": "0.0.0.0",
+  "HTTPPort": 8545,
+  "WSHost": "0.0.0.0",
+  "WSPort": 8586,
+  "APIModules": "connector,provider,eth,wallet",
```

Comment:
afaik `NODE_CONFIG` is not passed from desktop to status-go anymore. 
It's only possible with `LoginAccount` struct now.

Reply:
And it will be removed as soon as possible:
- https://github.com/status-im/status-desktop/issues/11435